bumble/gatt.py

```diff
@@ -259,63 +259,57 @@ class Characteristic(Attribute):
     See Vol 3, Part G - 3.3 CHARACTERISTIC DEFINITION
     '''
 
-    # Property flags
-    BROADCAST = 0x01
```

Comment:
I recommend adding a backward-compatible "hoisting" of the new names so that they can still be used via the older names.
Something like:
```
# Deprecated - only for backward compatibility. Use the `Properties` enum instead.
BROADCAST = Properties.BROADCAST
READ = Properties.READ
...
```
Unfortunately  I don't think there's a way to annotation class properties with `@deprecated`